ocs_ci/deployment/tnf.py

```
@@ -0,0 +1,543 @@
+        logger.info("Verifying TNF ODF deployment...")
+
+        try:
+            # Verify DRBD configuration
+            if not verify_drbd_configuration():
+                logger.error("DRBD configuration verification failed")
+                return False
+
+            # Verify DRBD status on both nodes
+            for node in self.node_info:
+                if not verify_drbd_status(node["name"]):
+                    logger.error(f"DRBD status check failed on node {node['name']}")
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)